resources/assets/js/videos/components/videoScreen/annotationPlayback.vue

```diff
@@ -125,6 +126,15 @@ export default {
                 this.updateGeometry(feature, time);
             });
         },
+        refreshAnnotation(annotation) {
```

Comment:
Since there also is a `refreshAnnotations` method here, I would rename the methods to `refreshSingleAnnotation` and `refreshAllAnnotations` to make the distinction more clear.